### PR TITLE
FHAC-565: NetworkAccessPointTypeCode fix

### PR DESCRIPTION
**Pull request by christophermay07**
- Unit test refactoring
- NetworkAccessPointTypeCode fix (setting appropriate value for IPs)

### CODE REVIEW

File: Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java [hunk 1]
```diff
@@ -78,38 +86,33 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
...
         if (isRequesting) {
-            assertEquals(getAuditTransforms().getServiceEventDisplayRequestor(), eventIdentificationType.getEventID()
-                .getDisplayName());
+            eventActionCode = getAuditTransforms().getServiceEventActionCodeRequestor();
+            eventIdCode = getAuditTransforms().getServiceEventIdCodeRequestor();
+            eventIdDisplayName = getAuditTransforms().getServiceEventDisplayRequestor();
         } else {
-            assertEquals(getAuditTransforms().getServiceEventDisplayResponder(), eventIdentificationType.getEventID()
-                .getDisplayName());
+            eventActionCode = getAuditTransforms().getServiceEventActionCodeRequestor();
```
Comment (vdmehta06) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java]: Isn't this incorrect? below details for eventActionCode,eventIdCode and eventIdDisplayName should be of  responder details not requestor details. For example using getServiceEventDisplayResponder()

**Comment by alameluchidambaram** [comment]
Reviewed PR and good with merge.

File: Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java (hunk 1 above)
Comment (christophermay07) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java]: Copy-paste error, I'm fixing now.  This is a serious problem, though -- it means that not one single one of our unit tests is properly testing the responder scenario.

Comment (alameluchidambaram) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java]: There is a ticket created in backlog to address responder scenario testcases.

**Comment by vdmehta06** [comment]
reviewed changes and green for merge

**Pull request merged**